src/io.jl

```diff
@@ -76,14 +78,21 @@ function read_datafile(filename::String)
 
     # Extract data arrays
     n_nodes = polydeg + 1
-    data = Array{Float64}(undef, n_nodes, n_nodes, n_elements, n_variables)
+
+    if ndims == 3
+      # Read 3d data
+      data = Array{Float64}(undef, n_nodes, n_nodes, n_nodes, n_elements, n_variables)
+    elseif ndims == 2
+      # Read 2d data
+      data = Array{Float64}(undef, n_nodes, n_nodes, n_elements, n_variables)
+    else
+      error("unsupported number of dimensions: $ndims")
+    end
     for v = 1:n_variables
       vardata = read(file["variables_$v"])
-      #=@show minimum(vardata), maximum(vardata)=#
-      @views data[:, :, :, v][:] .= vardata
+      @views data[.., v][:] .= vardata
```

Comment:
```suggestion
      @views data[.., :, v][:] .= vardata
```
To be consistent with Trixi.jl in that only the nodes are captured by the ellipsis operator?

Reply:
I don't see a reason to do that anymore since EllipsisNotation 1.0 is out. Which convention would you prefer here and in Trixi?

Reply:
I would prefer to use the same notation here as in Trixi to ensure consistency. That is, use `..` only to match the dimension-dependent nodes.